### ISSUE TITLE
Issue on browsing for file twice : OSX 12.1 with AIR 33.1.1.713

### ISSUE THREAD

**Issue by anthodb** [airsdk/Adobe-Runtime-Support#1509]
There is an issue while browsing for file on OSX 12.1 with AIR 33.1.1.713.
First time we use the Browse dialog for image files, it's working as expected.
Second time, during the same session the browse dialog is frozen.
Tested on multiple macs with different configurations, no workaround found.
In OSX 12, no issue.

**Comment by ajwfrost** [comment]
Thanks, yes we have a fix for this and are working on a release to update it..

**Comment by twhamilton** [comment]
This is a show stopper for our applications as well. Happy New Year from Apple.
Thanks in advance for the new release to fix this!

**Comment by ajwfrost** [comment]
Hi - fyi there's an updated AIR SDK to fix this now, currently available via the below link (or see the [Release Notes](https://airsdk.harman.com/release_notes) for details.
https://airsdk.harman.com/download/33.1.1.731
thanks

**Comment by stevenspeirs** [comment]
Hi. Any ETA on a new standalone runtime with the fix?
Thanks!

**Comment by myflashlab** [comment]
Awesome. @ajwfrost it now works flawlessly with Mac OS 12.1 👍

**Comment by Danik** [comment]
Thanks a lot for the fast response on this!

**Comment by pol2095** [comment]
FileFilter seems not working now, all files are selectable.

```
var txtFilter:FileFilter = new FileFilter("Images", "*.jpg;*.jpeg;*.gif;*.png");
var file:File = File.desktopDirectory;
file.browseForOpen("Please select a file...", [txtFilter]);
```

**Comment by ajwfrost** [comment]
@stevenspeirs I had wondered about that ... we appear to have a number of linked issues such as what @pol2095 mentions above plus other problems with file and directory browsing with 12.1 (I'm hoping that the issues are still existing with 12.2 since our test machine is now updated to that..!). Once we've done the next release, I'll get this loaded as a new shared runtime as well...

**Comment by twhamilton** [comment]
Another related issue that remains in 33.1.1.731 is that browseForDirectory is not behaving properly.
I behaves like browseForFile instead.

**Comment by anthodb** [comment]
We have clients reporting also the browseForDirectory is not working correctly on 12.2

**Comment by anthodb** [comment]
Do you have plan for a new release? The problem with browsing for directories or multiple files at once is really critical for our business. Thanks a lot

**Comment by ajwfrost** [comment]
Yes - updated release was posted yesterday, both as a new SDK version and also a new runtime version (you may need to just refresh the web page in case it's cached?) - 33.1.1.743 is the number you should see now on https://airsdk.harman.com/runtime

thanks

**Comment by pol2095** [comment]
Thanks, FileFilter work now.

**Comment by Ann-JS** [comment]
I am very happy to report that the latest Harman Air .743 Update has fixed all of the problems which I was having when  using JuiceBox Pro with the previous Harman build.
The Images Panel/Browse now lets me find and select multiple image files; 
and the Publish/Save/Browse button now allows one to set a new named location into which to Save a Gallery.
Image-population was quite slow the first time that I ran it immediately after installation of the new Harman Air Update but this has now returned to normal speedy loading with my second run.
I am running JBPro on an M1 MacBook Pro.

Thank you for fixing this issue,

Ann
